docs/api/graphql_queries.md

```diff
@@ -183,18 +207,50 @@ Response:
 
 ## Querying Locations
 
-!!! tip
+You can get the Location object from any Content item object by querying for `_location` or `_allLocations`.
+When you use `_location`, the API picks the Location to return based on the current SiteAccess, or returns the main one.
+
+```
+{
+  content {
+    folder (contentId: 133) {
+      _allLocations {
+        id
+      }
+    }
+  }
+}
+```
+
+Response:
 
-    To make use of enhanced Location handling, you can add the beta 3.0 version of [ezplatform-graphql](https://github.com/ezsystems/ezplatform-graphql/tree/3.0) to your project.
+```
+{
+  "data": {
+    "content": {
+      "folder": {
+        "_allLocations": [
+          {
+            "id": 132
+          },
+          {
+            "id": 133
+          }
+        ]
+      }
+    }
+  }
+}
+```
 
-    See [overview of the upcoming changes](https://github.com/ezsystems/ezplatform-graphql/pull/90).
+## Getting children of a Location
 
-To query a Location and its children, use the repository schema:
+To get a Location's children, query the `children` property of an `item` or `content` object:
```

Comment:
It would probably be worth mentioning that query fields are still preferred over that method :)

We may also want to remind that the location that is used to retrieve children is either automatic (if a content id was specified, or in case of a list of items), or manual (if a location id or url alias was used to find the item).